teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -179,10 +164,10 @@ private void initialise() {
     }
     genesisRoot = maybeRoot.get();
 
-    // get the validator
-    final Path slashProtectionPath = getSlashingProtectionPath(dataOptions);
+    // slashing protection won't be looked at, but it is required to create a validator loader.
+    // No files will be created in the path
     final SlashingProtector slashingProtector =
-        new SlashingProtector(new SyncDataAccessor(), slashProtectionPath);
+        new SlashingProtector(new SyncDataAccessor(), Path.of("."));
```

Comment:
Probably worth making `SlashingProtector` an interface so we can just supply a `FORBID_ALL` option here (validator exit shouldn't ask to sign any attestations or blocks so if it does, slashing protector should reject the request)

Reply:
hopefully i encapsulated pretty much what you're saying here.